ci/build/upload_build_info.sh

```diff
@@ -19,8 +22,10 @@ RAY_DIR=$(cd "${ROOT_DIR}/../../"; pwd)
 cd "${RAY_DIR}"
 
 cleanup() {
-  # Cleanup the directory because macOS file system is shared between builds.
-  rm -rf /tmp/bazel_event_logs
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    # Cleanup the directory because macOS file system is shared between builds.
+    rm -rf /tmp/bazel_event_logs
+  fi
```

Comment:
should we just do the remove on all systems? why change the behavior?

Reply:
also not changing the behavior, the existing implementation supposes to delete it only on mac

Reply:
ah, ok. you can split this part into a separate small PR if you want to get this particular change in.

Reply:
totally